### ISSUE TITLE
Accessibility Compliance

### ISSUE THREAD

**Issue by chrisray79** [evilC/AutoHotInterception#79]
Good day,

Would it be possible to review\modify the AutoHotInterception's code to make this add-in compliant to the WCAG 2.1 standards. Our department requires all applications to be accessible. Below you will find errors identified in AutoHotInterception following an accessibility assessment:

### Error Title & Description:
**1. Focus order unpredictable**
When you navigate through the application using the keyboard, the order of reading (focus) should follow the visual and logical order. In this case, the order of reading is unpredictable, it's the way the Software was coded. You may contact the vendors to address this issue in order for them to develop and design the product with a better accessibility approach.

**2. Screen reader not notified of change in state, such as the expansion/collapse of a section**
In Context Mode, If you unplug / re plug a device, the Screen Narrator is not notified of the change of state and will not read the changes. (Under KeyName & Info). I believe there is already a problem with that under the Known Issues (On GitHub), but it also affects the Screen Reader, unfortunately. There is no problem in Subscription mode.

**3. Keyboard Trap**
Cannot tab past a certain part of the page
When you navigate through the application using the keyboard, the order of reading (focus) should follow the visual and logical order. In this case, the order of reading is unpredictable and sometimes causes a Keyboard Trap, which you cannot get out of that component or element through the use of the keyboard.

**4. Elements on page in illogical order**
The tab order may follow an unexpected order because of the way the Software/Add-on is coded. Some users with disabilities such as low vision or blind may experience some difficulty navigating through the application with the use of a Speech Narrator. The focus of the keyboard should be placed immediately in the first field of the screen.

If you have any questions or comments, please let me know.

Thank you

**Comment by evilC** [comment]
Are you talking about the `Monitor.ahk` app here?
